tools-local/targetgroups.props

```diff
@@ -203,6 +203,13 @@
       <Imports>net462</Imports>
       <CompatibleWith>netstandard</CompatibleWith>
     </TargetGroups>
+    <TargetGroups Include="net471">
+      <TargetingPackNugetPackageId>Microsoft.TargetingPack.NETFramework.v4.7.1</TargetingPackNugetPackageId>
+      <NuGetTargetMoniker>.NETFramework,Version=v4.7.1</NuGetTargetMoniker>
+      <NuGetTargetMonikerShort>net471</NuGetTargetMonikerShort>
+      <Imports>net462</Imports>
```

Comment:
@weshaggard this is the only piece I'm not 100% sure I'm doing the right thing. Does that look correct?

Reply:
Ideally it would import net47 but we don't appear to have that one right now so this should be good enough.